### PR TITLE
decoupling example

### PR DESCRIPTION
**Pull request by sauravbhattacharya001**
blog post on decoupling

### CODE REVIEW

File: src/data/author.yaml [hunk 1]
```diff
@@ -17,4 +17,8 @@
 - id: snowfrog
   name: Philippe Vaillancourt
   bio: Phil likes to code. Phil likes to read. Phil likes to read about code. Occasionally, Phil also writes about code.
-  image: ../../src/images/authors/PhilVaillancourt.jpg
\ No newline at end of file
+  image: ../../src/images/authors/PhilVaillancourt.jpg
+- id: zalenix
+  name: Saurav Bhattacharya
+  bio: Saurav is interested in making well-desingned software. In his free time, he researches about software design, well-being practices, spritituality and Nondualism.
```
Comment (KyleMcMaster) [inline src/data/author.yaml]: typo: designed, spirituality

**Review by KyleMcMaster — CHANGES_REQUESTED** [review]
Some small typos but other than that it looks good! Thanks for the contribution!

**Review by KyleMcMaster — APPROVED** [review]
Looks good to me, :shipit: 

I'll let @ardalis merge this PR so he can deploy whenever he gets a chance.

**Comment by ardalis** [comment]
Good to go - sorry for delay

**Pull request merged**